generator/sbpg/targets/resources/sbp_messages_mod.rs

```diff
@@ -37,6 +37,21 @@ pub trait SBPMessage: SbpSerialize {
     }
 }
 
+pub trait RealMessage: SBPMessage + std::convert::TryFrom<SBP, Error = TryFromSBPError> + Clone + Sized {
```

Comment:
Not sure if `RealMessage` is the best name, but it's what it's called in the code generator

Reply:
`ConcreteMessage`? `KnownMessage`? `ReifiedMessage`?